src/components/formly.form.ts

```diff
@@ -63,18 +63,21 @@ export class FormlyForm implements OnInit  {
         }
 
         if (componentType.createControl) {
-          form.addControl(field.key, componentType.createControl(this.model[field.key] || '', field));
+          form.addControl(field.key, componentType.createControl(model[field.key] || '', field));
         } else {
-          form.addControl(field.key, new FormControl({ value: this.model[field.key] || '', disabled: field.templateOptions.disabled }, field.validation));
+          form.addControl(field.key, new FormControl({ value: model[field.key] || '', disabled: field.templateOptions.disabled }, field.validation));
         }
       }
 
       if (field.fieldGroup) {
         if (field.key) {
-          form.addControl(field.key, new FormGroup({}, field.validation));
+          const nestedForm = new FormGroup({}, field.validation);
```

Comment:
May I ask why? Because `nestedModel` is just a arbitrary object, whereas the first argument of `FormGroup` accepts only `FormControls` (`AbstractControl` to be more precise).